### ISSUE TITLE
User Management

### ISSUE THREAD

**Issue by skos-ninja** [DoESLiverpool/somebody-should#175]
To create a better way to create users and to be able to manage them better a system will need to be made. This issue should also be used to be referenced by others for us to know what should be added to the system

**Comment by DefProc** [comment]
Is this related to #45 or to the doorbot setup?

**Comment by madjake45** [comment]
Yes it's related to both really, it's also for helpful for any issues that arise in which we think it would be good to include that in the User Management.

**Comment by amcewen** [comment]
We should have a look at https://www.fab-manager.com/?lang=en to see if it covers the sort of things we want from an online system, and then think about switching over to that (and extending it to support the other things we'd want if need be)

**Comment by amcewen** [comment]
I wonder if fab-manager has somewhere that we can store the organisers-only info (passwords, process, etc.) that's currently the remaining bit of info on the wiki hosted on my VPS (so we can move it onto DoES-controlled infrastructure, as per #18 :-)  Don't suppose you noticed anything along those lines when you were having a poke around @DoESsean?

**Comment by DoESsean** [comment]
I didn't, but then I wasn't going into any great depth. I can have a more
detailed look today.

**Comment by jamieisboss** [comment]
The database for that is in postgres, if it has LDAP and Radius plugins, you'll save a lot of time as that can pretty much do network access, door access, file server/printer resources, github integration and all kinds of other things but they're the two things you really want for user management, then anything else to plug into them

**Comment by stevesparrow** [comment]
I think this is now all on Gitlabs ...

_Sent from my Asus P027 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by amcewen** [comment]
This isn't on Gitlabs, it's still something we haven't got a good system for.  I've got a fabmanager install now that we can poke around with and see how to modify to do what we need for this (and some of the other issues on administering DoES, members, bookings, etc.).

It'd be good to find a few people who want to help specify and build it - so if anyone wants to do(/learn) some Ruby coding, let me know.

I'll be leading the team pulling that together, but am pretty overloaded right now so it'll likely be a few weeks before there's much movement on it.

**Comment by skos-ninja** [comment]
@amcewen happy to help when the time comes

**Comment by amcewen** [comment]
I've killed off the FabManager install for now, as it was costing us money to keep running and it'll be nearer/after the move before I'll be able to do any more on it.  I've stashed the config files, etc. in somewhere-really-safe on Gitlab, so resurrecting it should be easier.
